include/UnicycleOptimization.h

```diff
@@ -122,6 +122,43 @@ class MaxAngle : public iDynTree::optimalcontrol::Constraint{
     }
 };
 
+class StepInEllipsoid : public iDynTree::optimalcontrol::Constraint{
+
+    FreeSpaceEllipse m_freeSpace;
+
+public:
+    StepInEllipsoid()
+        :iDynTree::optimalcontrol::Constraint(1,"StepInEllipsoid")
+    {
+        m_lowerBound.resize(1);
+        m_lowerBound(0) = 0.5;
```

Comment:
What does this lower bound value imply for this StepInEllipsoid? Could we pass it as a parameter?

Reply:
It is just a logical threshold. https://github.com/robotology/unicycle-footstep-planner/pull/40/files/76be3a8e8c361012f7e32d2f9a43f1c6b5e468d0#diff-e8850a66cc42fb85704b4cda559032a750523ba8ac36c1c54992e66530df16aaR157 returns either 0 or 1, so when the value is below 0.5 the constraint is not satisfied.

Reply:
Understood.